### ISSUE TITLE
Release to GitHub Registry was failed.

### ISSUE THREAD

**Issue by say8425** [JS-DevTools/npm-publish#26]
Hello. I maintained the [aws secrets manager actions](https://github.com/say8425/aws-secrets-manager-actions).

Some months ago, I tried to release to GitHub Registry with this NPM Publish in GH Action.
I tried both times but It was failed.
I can not find why it was failed. I share my GH Actions code and Workflows.

Can you help me?

#### Tried GH Action workflow codess

* https://github.com/say8425/aws-secrets-manager-actions/runs/1584302728?check_suite_focus=true

```yaml
# .github/workflows/release.yml
- name: Rlease to GitHub Registry    
  uses: JS-DevTools/npm-publish@v1   
  with:    
    token: ${{ secrets.GITHUB_TOKEN }}   
    registry: 'https://npm.pkg.github.com'   
    access:  ${{ format('@{0}', github.repository_owner) }}
```

* https://github.com/say8425/aws-secrets-manager-actions/runs/1338353950?check_suite_focus=true

```yaml
# .github/workflows/release.yml
- name: Rlease to GitHub Registry
  uses: JS-DevTools/npm-publish@v1
  with:
    token: ${{ secrets.GITHUB_TOKEN }}
    registry: 'https://npm.pkg.github.com'
```

### Raised GH Action Errors

Both of them raised a same error.

```
Error: Error: Unable to determine the current version of aws-secrets-manager-actions on NPM. 
npm view aws-secrets-manager-actions version exited with a status of 1.
npm ERR! code E404
npm ERR! 404 'aws-secrets-manager-actions' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it
npm ERR! 404 (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
npm ERR! 404 
npm ERR! 404  'aws-secrets-manager-actions@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2020-12-20T10_21_04_628Z-debug.log
    at Object.getLatestVersion (/home/runner/work/_actions/JS-DevTools/npm-publish/v1/src/npm.ts:57:13)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
ProcessError: npm view aws-secrets-manager-actions version exited with a status of 1.
npm ERR! code E404
npm ERR! 404 'aws-secrets-manager-actions' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it
npm ERR! 404 (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
npm ERR! 404 
npm ERR! 404  'aws-secrets-manager-actions@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2020-12-20T10_21_04_628Z-debug.log
    at normalizeResult (/home/runner/work/_actions/JS-DevTools/npm-publish/v1/node_modules/@jsdevtools/ez-spawn/lib/normalize-result.js:31:1)
    at ChildProcess.<anonymous> (/home/runner/work/_actions/JS-DevTools/npm-publish/v1/node_modules/@jsdevtools/ez-spawn/lib/async.js:79:1)
    at ChildProcess.emit (events.js:210:5)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
```

**Comment by JamesMessinger** [comment]
This has now been fixed by PR #19 and has been released as version 1.4.1

**Comment by say8425** [comment]
Thanks for your super fast fixing!
I'll try again release to GitHub Registry. Thanks!
